examples/supplementary/format_concat_args/rust-toolchain

```diff
@@ -0,0 +1,3 @@
+[toolchain]
+channel = "nightly-2025-02-20"
```

Comment:
```suggestion
channel = "nightly-2025-04-03"
```
There's a test to ensure that all examples use the same toolchain, and this is the toolchain they currently use.

Also, I think this is the toolchain that goes with the version of `clippy_utils` you are currently using.